bwa3_comp_ace/CfgWeapons.hpp

```diff
@@ -117,13 +203,21 @@ class CfgWeapons {
         ACE_barrelTwist = 304.8;
         ACE_barrelLength = 550;
         ACE_RailHeightAboveBore = 3.6;
+
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 252;
```

Comment:
ACE uses a mass of 218 for the MG5:
https://github.com/acemod/ACE3/blob/master/addons/realisticweights/CfgWeapons.hpp#L328